api/src/data.py

```diff
@@ -126,7 +126,7 @@ def cancel_job(job_id):
 
 # Last to not interfere with other routes
 @router.post("/job/{uuid}/{job_string}")
-def job(uuid: str, job_string: str, options: Optional[Dict[Any, Any]] = None):
+def runjob(uuid: str, job_string: str, options: Optional[Dict[Any, Any]] = None):
```

Comment:
Not sure I love the name `runjob` because it both runs a job and checks/returns the status of running jobs.
I think `job` works for both cases, but I'm open to something else or an argument for `runjob`.